server/service/src/sync/translations/invoice.rs

```diff
@@ -209,10 +209,9 @@ impl SyncTranslation for InvoiceTranslation {
             .find_one_by_name_id(&data.name_ID)?
             .map(|store_row| store_row.id);
 
-        let invoice_type = invoice_type(&data._type, &name).ok_or(anyhow::Error::msg(format!(
-            "Unsupported invoice type: {:?}",
-            data._type
-        )))?;
+        let invoice_type = invoice_type(&data._type, &name, data.mode.clone()).ok_or(
```

Comment:
At this stage, probably shoudl pass &data to invoice_type, if you didn't thought do you need to `clone()` mode ?